backend/lib/dotferno_web/components/layouts/app.html.heex

```diff
@@ -11,7 +11,7 @@
             width="80"
             height="80"
             alt="DotFerno logo"
-            class="w-12 h-14 md:w-10 md:h-16 logo-animation"
+            class="w-14 h-14 sm:w-16 sm:h-16 md:w-20 md:h-20 logo-animation"
```

Comment:
![Screenshot 2024-09-24 at 22 58 50](https://github.com/user-attachments/assets/ae667c41-968f-49ef-a398-fa428fada8bb)

Like this, right?